Switch fbcode builds of ExecuTorch and PyTorch to fbsource sleef

swolchok opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #11261

Stop using the fbcode-specific one.

Differential Revision: [D74208085](https://our.internmc.facebook.com/intern/diff/D74208085/)

**NOTE FOR REVIEWERS**: This PR has internal Meta-specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D74208085/)!

swolchok committed:
Switch fbcode builds of ExecuTorch and PyTorch to fbsource sleef

Stop using the fbcode-specific one.

Differential Revision: [D74208085](https://our.internmc.facebook.com/intern/diff/D74208085/)

**NOTE FOR REVIEWERS**: This PR has internal Meta-specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D74208085/)!

[ghstack-poisoned]

kimishpatel reviewed (approved):
propagating accept of the diff

swolchok committed:
Update on "Switch fbcode builds of ExecuTorch and PyTorch to fbsource sleef"


Stop using the fbcode-specific one.

Differential Revision: [D74208085](https://our.internmc.facebook.com/intern/diff/D74208085/)

**NOTE FOR REVIEWERS**: This PR has internal Meta-specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D74208085/)!

[ghstack-poisoned]
